extensions/mssql/src/features.ts

```diff
@@ -1279,10 +1279,25 @@ export class ExecutionPlanServiceFeature extends SqlOpsFeature<undefined> {
 			);
 		};
 
+		const isExecutionPlanXml = (executionPlanXml: string): Thenable<azdata.executionPlan.IsExecutionPlanXmlResult> => {
+			const params: contracts.IsExecutionPlanXmlParams = {
```

Comment:
given that the check for isExecutionPlan a string comparison with a constant value for MSSQL, I'd recommend doing that check inline here and avoid sending a message to STS.  This should allow removing the contracts and updating STS.  Also avoids a IPC message at runtime.